Pipfile

```diff
@@ -30,6 +30,7 @@ phonenumberslite = "==8.10.16"
 django-debug-toolbar = "==2.0"
 django-countable-field = {editable = true,git = "https://github.com/cityofaustin/django-countable-field"}
 django-flags = "*"
+wagtail-factories = "*"
```

Comment:
Can you add all the dependencies you're going to need to the Pipfile as well? I've got a change coming that's going to use Pipfile as the single source of truth. And maybe things like Faker can go into dev-dependencies.

Reply:
this is the only new dependency I added, it uses Faker as a dependency directly 

All the changes in requirements.txt are becuase I re-froze it, and pip sorts requirements.txt alphabetically by default 

FYI @briaguya this addresses your question about requirements.txt as well